frontend/src/pages/AdminPage.tsx

```diff
@@ -8,9 +9,19 @@ import LayoutWithNav from '../layouts/LayoutWithNav'
 const AdminPage: FunctionComponent = () => {
   return (
     <LayoutWithNav>
-      <div className="text-navy-800 flex flex-col space-y-2">
-        <Link to="/groups">Groups</Link>
-        <Link to="/users">Users (in construction)</Link>
+      <div className="bg-white max-w-5xl mx-auto border-l border-r border-gray-200 min-h-content">
```

Comment:
Nothing fancy, just making the page look half decent:

<img width="1065" alt="Screen Shot 2021-05-20 at 10 15 29 AM" src="https://user-images.githubusercontent.com/3411183/119021492-58044b80-b954-11eb-9464-c3530c89e9bc.png">